all data/src/mirror.py

```
@@ -603,6 +603,12 @@ def find_gifts(self, statuses):
                 threshold = 0.79
             else:
                 threshold = 0.75
+
+            # Similarly, it can mistake circular part of left side fusion UI as slash icon
+            if i == 'slash':
+                threshold = 0.79
+            else:
+                threshold = 0.75
```

Ruff (0.11.9)

Use ternary operator `threshold = 0.79 if i == 'slash' else 0.75` instead of `if`-`else`-block

Replace `if`-`else`-block with `threshold = 0.79 if i == 'slash' else 0.75`

(SIM108)